src/addressBook/index.js

```diff
@@ -0,0 +1,289 @@
+/**
```

Comment:
I think we may want to figure out if these are "model" files and where we want to formally put them. I'm not convinced `src/` is the right place for this file.

Reply:
Where should it be if not under `src/`?

Reply:
It will till be under `src` but I think we may want some other kind of sub-folder to contain these types of files. I just called out that I'm not sure that's where we ultimately want to keep these types of modules.

Reply:
We need a discussion on how to conceptualize these modules, to see if there's a pattern that we can formalize into something consistent.

Reply:
So yeah, I agree with you ^_^